backend/src/main/java/ai/verta/modeldb/utils/HandlerUtil.java

```diff
@@ -0,0 +1,59 @@
+package ai.verta.modeldb.utils;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.apache.logging.log4j.Logger;
+import org.jdbi.v3.core.Handle;
+
+public abstract class HandlerUtil {
+
+  protected String buildInsertQuery(Map<String, Object> rmValueMap, String entityName) {
+    // Created comma separated field names from keys of above map
+    String[] fieldsArr = rmValueMap.keySet().toArray(new String[0]);
+    var commaFields = String.join(",", fieldsArr);
+
+    // Created comma separated query bind arguments for the values
+    // based on the
+    // keys of
+    // above the map
+    // Ex: VALUES (:project_id, :experiment_id, :name) etc.
+    var bindArguments =
+        String.join(",", Arrays.stream(fieldsArr).map(s -> ":" + s).toArray(String[]::new));
+
+    return String.format(
+        "insert into %s (%s) values (%s) ", entityName, commaFields, bindArguments);
+  }
+
+  protected void buildUpdateQueryAndExecute(
+      Logger LOGGER,
+      Handle handleForTransaction,
+      long entityId,
+      Map<String, Object> rmValueMap,
+      String entityName) {
+    StringBuilder queryStrBuilder = new StringBuilder(String.format("UPDATE %s SET ", entityName));
+
+    AtomicInteger count = new AtomicInteger();
```

Comment:
Nit: we don't need an atomic counter as we won't be calling it in parallel.

Reply:
AtomicInteger is there because we used for loop as a lambda and in lambda if we used integer from out side then we have to make it AtomicInteger OR integer array with single value. but i have modified code with simple for loop and remove atomic. :)